packages/legacy/index.ts

```diff
@@ -1,3 +1,4 @@
 import { Tooltip } from "./src/Tooltip/Tooltip";
+import { Modal } from "./src/Modal/Modal";
```

Comment:
Is there a specific reason why you do `import ...; export ...` instead of just `export {Modal} from "./src/Modal/Modal"` ?

Reply:
https://github.com/dcos-labs/ui-kit/blob/master/packages/tabs/index.ts these are some examples.

Reply:
No reason. I can change these to just `export {Component} from "module"`

Reply:
That would be awesome, this will remove the problem of conflicts. and essentially does the same.